panels/templates/emails/panel_app_confirmation.txt

```diff
@@ -0,0 +1,5 @@
+{{ panelapplication.submitter.first_name }},
+
+We've recieved your application for a {{ c.EVENT_NAME }} panel.
```

Comment:
Good idea.  As you can probably tell, this is just a placeholder; the panels folks say that they'll send me the text of the real email later this week.  When that happens I'll make sure to use ``c.EVENT_NAME_AND_YEAR``.